src/config.rs

```diff
@@ -984,6 +984,34 @@ impl FontOffset {
     }
 }
 
+
+/// Modifications to glyph positions within their cells
+///
+/// By default the glyphs are located at the bottom of the cell which can be
+/// undesirable. This gives the user a way to shift where the glyphs are
+/// displayed in their cells.
+#[derive(Debug, Deserialize)]
+pub struct GlyphOffset {
+    /// Horizontal position
+    x: f32,
+    /// Vertical position
+    y: f32,
+}
+
+impl GlyphOffset {
+    /// Horizontal position
+    #[inline]
+    pub fn x(&self) -> f32 {
```

Comment:
I think making the properties `pub` would be the easiest thing. These methods can just go away then.